mcp-worker/src/consentScreen.ts

```diff
@@ -21,178 +21,454 @@ export function renderConsentScreen({
+                            display: block;
+                            max-width: 150px;
+                            margin: 1rem auto 1rem auto;
+                        }
+
+                        html, body {
+                            min-height: 100%;
+                            overflow: auto;
+                            background: 
+                                linear-gradient(29deg, rgba(109, 40, 217, 0.1) 0%, rgba(255, 255, 255, 0.5) 20%, rgba(255, 255, 255, 0) 23%),
+                                linear-gradient(305deg, rgba(29, 78, 216, 0.1) 0%, rgba(255, 255, 255, 0.5) 25.5%, rgba(255, 255, 255, 0) 26.5%);
+                            height: auto;
+                        }
+
+                        .consent-container {
+                            display: block;
+                            height: auto;
+                            width: auto;
+                        }
+
+                        .marketing-column {
+                            display: none;
+                        }
+
+                        .consent-column {
+                            display: block;
+                            flex: none;
+                            justify-content: normal;
+                            align-items: normal;
+                            padding: 1rem;
+                            background: none;
+                            position: static;
+                        }
+
+                        .consent-wrapper {
+                            display: block;
+                            max-width: none;
+                        }
+
+                        .consent-container-inner {
+                            margin-top: 1rem;
+                            margin-bottom: 2rem;
+                        }
+
+                        .footer {
+                            margin-bottom: 2rem;
+                        }
+                    }
+
+                    /* Remove gradients on small mobile screens */
+                    @media (max-width: 600px) {
+                        html, body {
+                            background: #ffffff !important;
+                        }
+                        
+                        .consent-column {
+                            padding: 0.5rem;
+                        }
                     }
                 </style>
             </head>
             <body>
-                <div class="container">
-                    <div class="logo">
-                        ${clientLogo?.length
-                            ? `<img src="${clientLogo}" alt="${clientName} logo" />`
-                            : ''}
+                <div class="consent-container">
+                    <!-- Marketing Column -->
+                    <div class="marketing-column">
+                        <div class="marketing-content">
+                            <img class="marketing-logo" src="https://devcycle.com/_next/static/media/logo.dark.9590d9ad.svg" alt="DevCycle" />
```

Comment:
don't use this url - it's not guaranteed to be around.
You should use the brand asset paths. 

https://devcycle.com/assets/brand/White-logo-horizontal.svg